Library/Formula/fpc.rb

```diff
@@ -25,11 +25,15 @@ def install
     system "make", "install", "PP=#{fpc_compiler}", "PREFIX=#{prefix}"
 
     bin.install_symlink lib/"#{name}/#{version}/ppcx64"
+
+    # generate a default fpc.cfg to facilitate search paths
+    system "./utils/fpcmkcfg/fpcmkcfg", "-p", "-d", "basepath=#{Formula["fpc"].opt_lib}/fpc/" + `./compiler/ppc1 -iW`.chomp, "-o", "#{Formula["fpc"].opt_prefix}/etc/fpc.cfg"
```

Comment:
Actually since this _is_ the `fpc` formula, you can just use `lib` and `prefix`.